cmd/internal/distributor/distributor.go

```diff
@@ -40,10 +40,13 @@ import (
 const maxSigs = 100
 
 var (
-	counterCheckpointUpdateRequests = promauto.NewCounter(prometheus.CounterOpts{
-		Name: "distributor_update_checkpoint_request",
-		Help: "The total number of requests to update a checkpoint",
-	})
+	counterCheckpointUpdateRequests = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "distributor_update_checkpoint_request",
```

Comment:
Can you also update the corresponding success counter to also be scoped by witness?